src/App.jsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import { Route, Routes, BrowserRouter as Router } from 'react-router-dom';
-import HomePage from './pages/HomePage';
-import LoginPage from './pages/LoginPage';
+import HomePage from './pages/HomePage/HomePage';
+import LoginPage from './pages/Login/LoginPage';
 import './App.css';
+import Dashboard from './pages/Dashboard/Dashboard';
+import Profile from './pages/Profile/Profile';
+import Surveys from './pages/Surveys/Surveys';
+import Data from './pages/Data/Data';
+import ImportCSV from './pages/ImportCSV/ImportCSV';
+import AddData from './pages/AddData/AddData';
 
 function App() {
   return (
-    <Router>
-      <Routes>
-        <Route exact path="/" element={<HomePage />} />
-        <Route exact path="/login" element={<LoginPage />} />
-      </Routes>
-    </Router>
+    <React.StrictMode>
+      <Router>
+        <Routes>
+          <Route path="/" element={<Dashboard />} />
+          <Route exact path="/homepage" element={<HomePage />} />
```

Comment:
we don't need both a `HomePage` as well as a `Dashboard`, they're essentially the same in this app

let's get rid of the duplicate `HomePage` and just route `/` to `Dashboard` like it currently is